### ISSUE TITLE
Upgrade main dependencies in packages/client and packages/components

### ISSUE THREAD

**Issue by OAGr** [foretold-app/foretold#871]
bs-css: to 11.0.0+
gentype: to 3.6.0+
bs-reform: to 9.5.0+
react-apollo: to 3.1.3+
reason-apollo: to 0.17.0+
rationale: to 0.2.0+

**Comment by RomanGalochkin** [comment]
Here I have upgraded "bs-css", "gentype" without any problems. But to upgrade "bs-reform" I had to rewrite all forms to use "reason-apollo-hooks" to support new version of "bs-reform" 9.5.0 and to leave "reason-apollo" and "reason-apollo-hooks" at the same time. There was not and there will not be another way of doing that since all third-party libraries go to "react hooks". But now, there is new not a huge issue with "prefetching of previous data". For instance, when someone creates new measurable we need to refresh meaurables list. "Bs-reform" supports the "prefetching" but it is not compatible with previous version of "reason-apollo" which does not use hooks at all. So here we need to find a way of doing that either we could make a refactoring other queries either we could turn off the cache for some number of queries.
